workloads/free5gc/ueransim/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: ueransim
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: this package represents ueransim NFs, which are required to perfirm E2E conn testing
```

Comment:
As the description states, it is indeed a test tool. With that, it prob should be stored somewhere more relevant.

Reply:
@electrocucaracha @efiacor It is a simulated RAN, it is designed to test the end to end connectivity. For UERANSIM, a core network (Free5gc or OAI Core) is indeed needed for end to end testing and vice versa. 

So I would still suggest to keep it in an individual folder rather than common or tool. Also we can use OAI RAN to test free5GC. But for this we need to match some configuration parameters and this we can do it later.

Reply:
I guess it's a matter of opinion but I have seen things get quite messy in the past when we start mixing simulators/test tools in with "production" code. I would clearly separate the 2 so that it's clear to the end user.

Reply:
@efiacor I understand your point too, in this case I am fine with common or tools directory.

Reply:
`tools` seems the right place to me.

Reply:
@efiacor what about adding a Notice in the README.md file saying that this package is used for testing purposes and not recommending for production.

Reply:
@electrocucaracha , yes I think that would be a good idea. Anything that avoids confusion for the user is helpful really.